src/components/movies/MovieDetail.vue

```diff
@@ -1,26 +1,136 @@
 <script>
 import { defineComponent } from 'vue';
+import { mapActions } from 'pinia';
+import { mainStore } from '@/store/index.js';
+
+import BaseTag from '@/components/base/BaseTag.vue';
 
 export default defineComponent({
+  components: {
+    BaseTag,
+  },
   props: {
     movie: {
       type: Object,
       required: true,
     },
   },
+  computed: {
+    imageStyles() {
+      let backgroundPosition;
+
+      if (this.movie.id === 1) backgroundPosition = '50% 30%';
```

Comment:
Images used for movies description are not anyhow standardised, like having same dimensions or focus point. So I did such thing just to please myself with (which is only my opinion) specific positioning of each image. I guess we can't really mathematically nor logically set positioning without standardised images when it's purely based on it's content. 

Otherwise I could just skip this and center all the background images, but in some cases that resulted with having our posters' blank areas in the center of their container.

Reply:
Sure. Here is an example of my positioning:
<img width="1378" alt="image" src="https://user-images.githubusercontent.com/66744401/188464162-88cdf2eb-a513-41df-a3dd-4c400273a1cc.png">

And the centered positioning:
<img width="1374" alt="image" src="https://user-images.githubusercontent.com/66744401/188464377-bd1c8c8d-0652-4d65-bfde-7491c1b48dd5.png">

This is actually the only "blank space" case to be honest, but others tend to not make sense in their own way, like showing characters legs instead of torso / portrait:
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/66744401/188464946-7b0e87bf-7fa9-464f-b775-933d8f4f694c.png">
<img width="1370" alt="image" src="https://user-images.githubusercontent.com/66744401/188464984-2ed46611-d6d8-45f0-bd33-ed3e86562ab4.png">

And I think the designs are clear about the image height has to fit the leftside text height.

Reply:
Aaand it already uses `background-size: cover;` 😅

Reply:
👍 Does it have to be so wide? If image is vertical maybe it would make sense to make it more narrow? With `background-size: contain`

Reply:
Had a look at designs and I guess it can't

Reply:
So one thing that i'd suggest here is to extract this configuration logic to a separate file maybe as an object with movie id as a key.

Reply:
I like the idea. To be honest in the real project (not a sandbox one) I would just start a discussion about that and ask designers if we can tweak it all a little bit so it doesn't create such fancy problem. Anyway, let's go with separate file with the object as you said  🚀 👌